docs/use-cases/one-to-many.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "How to configure multiple similar machines"
+linkTitle: "Configure many machines"
+weight: 40
+type: "docs"
+tags: ["data management", "data", "services"]
+images: ["/platform/fleet.svg"]
+description: "How to configure multiple similar or identical machines with fragments."
+---
+
+Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ for copying parts of a configuration file to multiple machines, as well as for modifying just some sections of a config if your machines are similar but not identical.
+
+When you update a fragment, it updates the configurations of all machines that use that fragment.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Create a configuration fragment](#create-a-fragment)
+1. [Apply a fragment to multiple machines](#add-a-fragment-to-multiple-machines)
+1. [Modify an otherwise identical configuration](#modify-a-fragment)
+
+{{< /alert >}}
+
+For information on provisioning many machines, see [Provisioning](/fleet/provision/).
+
+## Create a fragment
+
+{{< table >}}
+{{% tablestep link="/configure/" %}}
+{{<imgproc src="/use-cases/one-to-many/config.png" resize="700x" class="fill alignleft" style="max-width: 250px" declaredimensions=true alt="Configuration builder UI">}}
```

Comment:
This is an interesting approach but the screenshots are too small to be useful. If Cristina can create something that's great, if not I think use the full screenshots and add them in each step rather than to the side. Like in https://docs-test.viam.dev/3201/use-cases/configure/.

Reply:
Alternatively what we could do is keep these as small screenshots and make them clickable to expand

Reply:
https://github.com/viamrobotics/docs/pull/3206 adds functionality for it either way